mtconnect-client-impl/src/main/java/io/github/protocol/mtconnect/client/vertx/MtConnectClient.java

```diff
@@ -0,0 +1,16 @@
+package io.github.protocol.mtconnect.client.vertx;
+
+import io.github.protocol.mtconnect.client.api.MtConnectClientConfiguration;
+import io.github.shoothzj.http.client.facade.HttpClient;
+import io.github.shoothzj.http.client.facade.HttpClientFactory;
+
+public class MtConnectClient {
```

Comment:
did u forget `@Getter` ?

Reply:
I think client needs to add other methods, like start close, etc